etcdmgmt/etcd-mgmt_test.go

```diff
@@ -20,10 +48,11 @@ func TestStartETCD(t *testing.T) {
 	etcdCtx, err := StartETCD()
 	tests.Assert(t, err == nil)
 	etcdCtx.Kill()
+	etcdCtx.Wait()
 }
 
 func TestWriteETCDPidFile(t *testing.T) {
-	cmd := exec.Command("etcd")
+	cmd := formETCDCommand()
```

Comment:
after this you should also check whether cmd command is properly formed or its giving error.